avocado/utils/distro.py

```diff
@@ -349,6 +349,43 @@ class DebianProbe(Probe):
     CHECK_FILE_DISTRO_NAME = 'debian'
 
 
+class UbuntuProbe(Probe):
+
+    """
+    Simple probe for Ubunt systems in general
+    """
+
+    CHECK_FILE = '/etc/os-release'
+    CHECK_FILE_CONTAINS = 'ubuntu'
+    CHECK_FILE_DISTRO_NAME = 'Ubuntu'
+
+    def get_distro(self):
```

Comment:
@PraveenPenguin have you checked if it wouldn't be possible to just use https://avocado-framework.readthedocs.io/en/64.0/api/utils/avocado.utils.html#avocado.utils.distro.Probe.CHECK_VERSION_REGEX instead of this implementation? This regex should return either 1 or 2 groups, for the version and release.

Reply:
@clebergnu ah sure , thanks for pointing on this i did not check this